src/tests/test_time_travel.py

```diff
@@ -37,3 +39,36 @@ def test_sleep_changing_today():
         
         time.sleep(3600)
         assert datetime.today() == datetime.fromtimestamp(3600)
+
+
+def test_select_no_timeout():
```

Comment:
I do want some tests with select integration with other patchers.
They test things tested in the unit-test but integrated with the other patchers